IpcSubscribers/State.cs

```diff
@@ -186,6 +186,22 @@ public static FuncProvider<int, uint, int> Provider(IDalamudPluginInterface pi,
         => new(pi, Label, (a, b) => (int)api.UnlockState(a, b));
 }
 
+/// <inheritdoc cref="IGlamourerApiState.IsUnlocked"/>
+public sealed class IsUnlocked(IDalamudPluginInterface pi)
+    : FuncSubscriber<int, uint, (GlamourerApiEc, bool?)>(pi, Label)
+{
+    /// <summary> The label. </summary>
+    public const string Label = $"Glamourer.{nameof(IsUnlocked)}";
+
+    /// <inheritdoc cref="IGlamourerApiState.IsUnlocked"/>
+    public new (GlamourerApiEc, bool?) Invoke(int objectIndex, uint key = 0)

```

Comment:
I'd prefer if the wrapper is `GlamourerApiEc Invoke(int objectIndex, uint key, out bool isLocked, out bool canUnlock)`. Also no need for default value for key, doesn't make sense for this method.